ansible/roles/gateway-launcher/defaults/main.yml

```diff
@@ -3,6 +3,7 @@
 kamailio_host: 127.0.0.1
 kamailio_sip_secret: "1234"
 kamailio_sip_account_name_prefix: mediagw
+kamailio_enable_tls: "{{ lookup('env','ENABLE_TLS') | bool | default('false') }}"
```

Comment:
It seems that `lookup('env','...')` is always defined, so `default` catching should not be set